### PR TITLE
Related 60% test coverage

### PR DESCRIPTION
**Pull request by dootMaster**
Changed a lot of CSS. Let me know if it looks okay.

Modal still needs work.

### CODE REVIEW

**Comment by dootMaster** [comment]
Achieved 60% statement/line coverage total from contributions of Related widget tests.

**Comment by pgoggles** [comment]
On the gitignore you deleted the line that blocks the config from getting committed, the line should be:
**/config.js

This means block any config.js files that are one level under the root, which ours is, as our config.js is located at:
server/config.js

File: server/config.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const API_KEY = '5832a808f497f51e1f7b9df78835c3bb21451632';
+const API_KEY = '7881bab7e75759a23f7bfea70bb3ecd6ca1f2918';
```
Comment (pgoggles) [inline server/config.js:1]: You don't want this committed

Comment (dootMaster) [reply server/config.js:1]: good catch

**Pull request closed without merging**